### PR TITLE
how-to-contribute

### PR DESCRIPTION
**Pull request by lucasborin**

### CODE REVIEW

File: pages/how-to-contribute.md [hunk 1]
```diff
@@ -4,25 +4,179 @@
...
+
+The `get_cut` method must return an instance of the class under test.
+
+```abap
+METHOD get_cut.
+    result ?= NEW y_check_prefer_is_not( ).
+ENDMETHOD.
+```
+
+The `get_code_with_issue` method must return a snippet of code which raises the check.
```
Comment (eugen-guenther-sap) [inline pages/how-to-contribute.md:105]: The code snipped must be syntactical correct.

**Comment by eugen-guenther-sap** [comment]
maybe we should add a line about, that one check class should do just one thing at a time and not 10 checks in 1.

File: pages/how-to-contribute.md (hunk 1 above)
Comment (lucasborin) [reply pages/how-to-contribute.md:105]: ok. I added the ... as a comment.

**Comment by lucasborin** [comment]
Added. Could you please check agian?

**Review by eugen-guenther-sap — APPROVED** [review]

**Comment by estevao-schultz-neto-SAP** [comment]
I was going to review it today (had no time so far). But seems that you guys already pushed it into master? Should I review it anyway by creating a new pull-request ?

**Pull request merged**